web/src/utils/Context.ts

```diff
@@ -79,6 +79,17 @@ function ContextMerger<T extends {}>(
 const AuthKey = "autobrr_user_auth";
 const SettingsKey = "autobrr_settings";
 const FilterListKey = "autobrr_filter_list";
+const IncognitoKey = "autobrr_incognito";
+
+const getInitialIncognitoState = (): boolean => {
+  try {
+    const stored = localStorage.getItem(IncognitoKey);
+    return stored ? JSON.parse(stored) : false;
+  } catch (e) {
+    console.error(`Failed to read ${IncognitoKey} from localStorage`, e);
+    return false;
+  }
+};
```

Comment:
Could we merge this into the `SettingsContext` to have a single place for such settings/states?

Reply:
Should be good now I think